integration-tests/src/test/java/io/testerra/test/pretest_status/GenerateSimpleTestStatusTest.java

```diff
@@ -0,0 +1,51 @@
+package io.testerra.test.pretest_status;
+
+import eu.tsystems.mms.tic.testframework.execution.testng.AssertCollector;
+import eu.tsystems.mms.tic.testframework.execution.testng.OptionalAssert;
+import eu.tsystems.mms.tic.testframework.report.Status;
+import eu.tsystems.mms.tic.testframework.testing.TesterraTest;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+public class GenerateSimpleTestStatusTest extends TesterraTest {
+
+    private final String SKIPPED_EXCEPTION_MESSAGE = String.format("Test %s.", Status.SKIPPED.title);
+
+    @Test
+    public void test_Passed() {
+        Assert.assertTrue(true, "Creating TestStatus 'Passed'");
```

Comment:
This cannot work as intended, however intended it was: `assertTrue` does _nothing_ when `condition` is `true` (so the String parameter is wasted).

(FWIW: You can add a message to a passing test via post processing, but I currently cannot recall how it was done.)

Reply:
Well, I think it was used as documentation. I removed it.